### PR TITLE
Checking null values

### PR DESCRIPTION
**Pull request by bronx**
@nykolaslima

### CODE REVIEW

File: src/main/java/com/vivareal/logger/KeyValueLoggerDataBuilder.java [hunk 1]
```diff
@@ -42,13 +42,14 @@ protected KeyValueLoggerDataBuilder(Logger logger, String separator, Object obje
     
     @Override
     public LogDataConjunction value(Object value) {
-	keyValueMap.put(currentKey, value.toString());
+	keyValueMap.put(currentKey, (value != null ? value : "null").toString());
```
Comment (nykolaslima) [inline src/main/java/com/vivareal/logger/KeyValueLoggerDataBuilder.java]: Do we really want to log "null" for null values?

Comment (bronx) [inline src/main/java/com/vivareal/logger/KeyValueLoggerDataBuilder.java]: Sim.

Comment (nykolaslima) [inline src/main/java/com/vivareal/logger/KeyValueLoggerDataBuilder.java]: So go for it! Looks good to me.

**Comment by nykolaslima** [comment]
:+1:

**Comment by nykolaslima** [comment]
@bronx dont forget the vr-logger package name to do the release

**Pull request merged**